.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
+daysUntilClose: 7
```

Comment:
what's our general policy? if there is none, perhaps we should have one written up to be implemented across all libraries?

Reply:
starting one here https://paper.dropbox.com/doc/Libraries-Stale-Policy--AZW~MXAQ_Pn~fUKEZhq9vYbYAg-tiUzSGiTErmeXLWEdXNbW